CMakeLists.txt

```diff
@@ -1,19 +1,21 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 3.5)
 PROJECT(openpose-plus)
 
-SET(CMAKE_CXX_STANDARD 14)
+SET(CMAKE_CXX_STANDARD 17)
 
 INCLUDE_DIRECTORIES(${CMAKE_SOURCE_DIR}/include)
 INCLUDE(cmake/3rdparty.cmake)
 
 OPTION(WITH_TRACE "Build with tracing library." OFF)
 OPTION(BUILD_FAKE "Build fake target without CUDA libraries." OFF)
 
-IF(ENABLE_TRACE)
+IF(WITH_TRACE STREQUAL "ON")
```

Comment:
sometimes people would also use `-DWITH_TRACE=1` or `-DWITH_TRACE=on` instead of `-DWITH_TRACE=ON`

Reply:
What about : 
`IF(NOT WITH_TRACE STREQUAL "OFF")`.

Reply:
I just checked cmake official document:
https://cmake.org/cmake/help/latest/command/if.html#condition-syntax

The
```cmake
if(<constant>)
```

syntax is actuall well documented:

"True if the constant is 1, ON, YES, TRUE, Y, or a non-zero number. False if the constant is 0, OFF, NO, FALSE, N, IGNORE, NOTFOUND, the empty string, or ends in the suffix -NOTFOUND. Named boolean constants are case-insensitive. If the argument is not one of these specific constants, it is treated as a variable or string and the following signature is used."

So I think it's OK to leave it as it was.

Reply:
OK. Thanks for your suggestion.